cherryontech/jupiter-jumpers
# Update layout

## Description

Updates the layout for the website:
1) Adds a left and right sidebar
2) Only shows the top navigation bar when there isn't an authenticated user
3) Adds a menu of links on the left sidebar
4) Adds a dropdown menu (authenticated view) where the user has their avatar and username (includes the logout)

## Related Issue

## Related Story Card

[SCRUM-29](https://cherryontech-jupiter-jumpers.atlassian.net/browse/SCRUM-29)

## Type of Changes
Styling Update

## Acceptance Criteria

- [X]     Given I am on the home screen or another part of the app 
- [X]     When I look at my dashboard 
- [X]     Then I can see my "Monthly View" page

## Update Screenshots

_Provide screenshots of the view before and after the changes._

### Before
![Screenshot 2024-05-13 at 9 01 39 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/7b64d0ee-ffcb-40a4-baa3-14d9f705fab6)

### After
![Screenshot 2024-05-13 at 8 59 35 PM](https://github.com/cherryontech/jupiter-jumpers/assets/47403135/baf25d3b-6eb3-43cb-8054-2ea26181e9d1)

## Testing Instructions

1. When logged out, ensure you can still see the top navigation bar (with logo on the left, sign in/register on the right)
2. When logged in, ensure the navbar is gone.
3. Try clicking "Calendar" and "About" in the left menu; ensure the links turn blue when they've been selected and that the appropriate content loads in the 'main' (middle) view.
4. Try logging a mood (to ensure it didn't break)
5. Try opening the drop-down menu in the user menu (top right, where the avatar is)
6. Try logging out; confirm you can see the navigation bar again.

## Learnings (Optional)

_Document any things you learned or 'gotchas' you experienced._
